lib/TwoFactorAuth.php

```diff
@@ -79,7 +79,7 @@ public function getCode(string $secret, ?int $time = null): string
         $value = unpack('N', $hashpart);                                                   // Unpack binary value
         $value = $value[1] & 0x7FFFFFFF;                                                   // Drop MSB, keep only 31 bits
 
-        return str_pad((string) ($value % 10** $this->digits), $this->digits, '0', STR_PAD_LEFT);
+        return str_pad((string)($value % 10 ** $this->digits), $this->digits, '0', STR_PAD_LEFT);
```

Comment:
good spot on the missing space but I find a space after the type coercion much more readable

Reply:
This was also PHP-CS-Fixer.